openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -63,18 +62,26 @@ def get_settings_from_context(self, context):
             .get("publish", {})
             .get(self.__class__.__name__, {})
         )
+        get_from_context = False
+        if kitsu_note.get("set_status_note") != self.set_status_note:
+            self.set_status_note = kitsu_note["set_status_note"]
+            get_from_context = True
         if (
-            kitsu_note.get("set_status_note") != self.set_status_note,
             kitsu_note.get("note_status_shortname")
-            != self.note_status_shortname,
-            kitsu_note.get("status_change_conditions")
-            != self.status_change_conditions,
+            != self.note_status_shortname
         ):
-            self.set_status_note = kitsu_note["set_status_note"]
             self.note_status_shortname = kitsu_note["note_status_shortname"]
+            get_from_context = True
+        if (
+            kitsu_note.get("status_change_conditions")
+            != self.status_change_conditions
+        ):
             self.status_change_conditions = kitsu_note[
                 "status_change_conditions"
             ]
+            get_from_context = True
+
+        if get_from_context:
```

Comment:
For a more explicit logging, I'd recommend to change it to `settings_from_context = []`, append each one has been loaded from context and print them in the log.

Reply:
Updated in https://github.com/Tilix4/OpenPype/pull/78/commits/3c9334b3c05ae9a718e9a18a13b390bf12a2d504